CONTRIBUTING.md

```diff
@@ -182,6 +182,98 @@ components/
     my-component.spec.js
 ```
 
+#### Spec File Setup
+
+Each component will require a DOM-like environment in order to be queried for the custom element and, thus assert on the component details.
+
+The following is an example of a spec file, with imports, a global mock (`fetch`), and the `describe`/`it` blocks.
+
+```js
+import { expect } from "@esm-bundle/chai";
+import "./my-custom-element.js";
+
+window.fetch = function () {
+  return new Promise((resolve) => {
+    resolve(new Response(JSON.stringify({ foo: "mock response" })));
+  });
+};
+
+describe("Components/My Custom Element", () => {
+  it("should do something expected", () => {
+    expect("something").equal("some" + "thing");
+  });
+});
+```
+
+#### Testing the Component
+
+With the spec file foundation, the custom element can now be created and applied to the mock DOM environment. To do this, we use the Web Test Runner (WTR) framework which provides the scaffolding to test Web Componets in a browser environment.
+
+The WTR and the full setup is covered in greater detail within [the GreenwoodJS test runner docs](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/). In an attempt to not duplicate those docs, an outline has been provided here, but please refer to those docs for a full user guide.
+
+1. **[Setup](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#setup)**
+
+   Install dependencies and configure WTR. This project has that covered already! ✅ (Let's chat if there becomes a need to reconfigure the current setup.)
+
+1. **[Usage](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#usage)**
+
+   Within a traditional `before` block, create the custom element, and append to the DOM. See the example below. 👇
+
+1. **[Static Assets](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#static-assets)**
```

Comment:
by the time these docs go live, we'll be on our new domain, so you can safely replace all occurrences of `https://super-tapioca-5987ce.netlify.app` with `https://www.greenwood.dev`